roles/installer/defaults/main.yml

```diff
@@ -236,6 +236,7 @@ _init_projects_container_image: quay.io/centos/centos:stream9
 
 create_preload_data: true
 
+replicas: "1"
```

Comment:
do we really need a default replica here? Maybe set it to an empty string?

Reply:
should we leave this in and remove the defaults below and set to empty string?

Reply:
we shouldnt have default for web_replicas and task_replicas right?

Reply:
I am of this opinion @TheRealHaoLiu and that just replicas should be one. if you care about scaling these replicas separately, they are there for you to do so.